Marlin/src/lcd/ultralcd.cpp

```diff
@@ -1221,6 +1248,57 @@ void lcd_quick_feedback(const bool clear_buttons) {
     END_MENU();
   }
 
+  #if ENABLED(LIN_ADVANCE)
+    void lcd_linadv_menu(){
+      START_MENU();
+      MENU_BACK(MSG_BACK);
+      #if EXTRUDERS == 1
+        MENU_ITEM_EDIT(float52, MSG_ADVANCE_K, &planner.extruder_advance_K[0], 0, 999);
+      #elif EXTRUDERS > 1
+        MENU_ITEM_EDIT(float52, MSG_ADVANCE_K " " MSG_E1, &planner.extruder_advance_K[0], 0, 999);
+        MENU_ITEM_EDIT(float52, MSG_ADVANCE_K " " MSG_E2, &planner.extruder_advance_K[1], 0, 999);
+        #if EXTRUDERS > 2
+          MENU_ITEM_EDIT(float52, MSG_ADVANCE_K " " MSG_E3, &planner.extruder_advance_K[2], 0, 999);
+            #if EXTRUDERS > 3
+              MENU_ITEM_EDIT(float52, MSG_ADVANCE_K " " MSG_E4, &planner.extruder_advance_K[3], 0, 999);
+                #if EXTRUDERS > 4
+                  MENU_ITEM_EDIT(float52, MSG_ADVANCE_K " " MSG_E5, &planner.extruder_advance_K[4], 0, 999);
+                #endif // EXTRUDERS > 4
+              #endif // EXTRUDERS > 3
+        #endif // EXTRUDERS > 2
+#endif // EXTRUDERS > 1
+
+      END_MENU();
+    }
+  #endif
+
+  #if DISABLED(NO_VOLUMETRIC)
+    void lcd_filament_width_menu(){
+      START_MENU();
+      MENU_BACK(MSG_BACK);
+        MENU_ITEM_EDIT_CALLBACK(bool, MSG_VOLUMETRIC_ENABLED, &parser.volumetric_enabled, planner.calculate_volumetric_multipliers);
+
+        if (parser.volumetric_enabled) {
+          #if EXTRUDERS == 1
+            MENU_MULTIPLIER_ITEM_EDIT_CALLBACK(float43, MSG_FILAMENT_DIAM, &planner.filament_size[0], 1.5f, 3.25f, planner.calculate_volumetric_multipliers);
+          #else // EXTRUDERS > 1
+            MENU_MULTIPLIER_ITEM_EDIT_CALLBACK(float43, MSG_FILAMENT_DIAM " " MSG_ACTIVE, &planner.filament_size[active_extruder], 1.5f, 3.25f, planner.calculate_volumetric_multipliers);
```

Comment:
This is not compatible with translations, because different languages use different word orders.